docs/readme.md

```diff
@@ -94,7 +94,7 @@ In order to extend the Lighthouse configuration programmatically, you need to pa
       'first-meaningful-paint',
       'speed-index-metric',
       'estimated-input-latency',
-      'first-interactive',
+      'first-cpu-idle',
       'consistently-interactive',
```

Comment:
should probably update `consistently-interactive`, `speed-index-metric`, and kill `estimated-input-latency` if we're updating these now, or could just wait for when we really kill EIL :)